src/GridView/GridView.js

```diff
@@ -1,32 +1,54 @@
-import React, { Component } from 'react';
-import DonutChart from './DonutChart'
-import HierarchyStepper from './HierarchyStepper'
-import SearchBar from './../Utils/SearchBar'
+import React, {Component} from 'react';
+import {connect, PromiseState} from 'react-refetch';
+import LoadingAnimation from './../Utils/LoadingAnimation';
+import ErrorMessage from './../Utils/ErrorMessage';
+import DonutChart from './DonutChart';
+import HierarchyStepper from './HierarchyStepper';
+import SearchBar from './../Utils/SearchBar';
 import CardGrid from './CardGrid';
 
-class GridView extends Component {
-
-
-	componentDidMount() {
-		this.props.datasource.getEntities(3, this.getEntities);
-	}
 
-	getEntities(entities) {
-		this.setState({
-			entities: entities
+class GridView extends Component {
+	
+	getChildEntityTypes(parentEntityTypeId, hierarchy) {
```

Comment:
Because this appears in the context of GridView it makes sense to explicitly mention the child-parent dependency.